appservice/src/createAppService/AppInsightsCreateStep.ts

```diff
@@ -44,16 +45,42 @@ export class AppInsightsCreateStep extends AzureWizardExecuteStep<IAppServiceWiz
                     wizardContext.appInsightsComponent = await client.components.createOrUpdate(rgName, aiName, { kind: 'web', applicationType: 'web', location: appInsightsLocation });
                     const createdNewAppInsights: string = localize('createdNewAppInsights', 'Successfully created Application Insights resource "{0}".', aiName);
                     ext.outputChannel.appendLog(createdNewAppInsights);
+                } else if (pError.errorType === 'AuthorizationFailed') {
+                    if (!wizardContext.advancedCreation) {
+                        const createAppInsightsNotAvailable: string = localize('appInsightsNotAvailable', 'Skipping Application Insights resource because you do not have permission to create one in this subscription.');
+                        ext.outputChannel.appendLog(createAppInsightsNotAvailable);
```

Comment:
```suggestion
                        const appInsightsNotAuthorized: string = localize('appInsightsNotAuthorized', 'Skipping Application Insights resource because you do not have permission to create one in this subscription.');
                        ext.outputChannel.appendLog(appInsightsNotAuthorized);
```